wicker/core/storage.py

```diff
@@ -392,12 +405,16 @@ def _get_column_concatenated_bytes_files_path(
         """
         if self.store_concatenated_bytes_files_in_dataset:
             if dataset_name is None:
+                # I hate exceptions but they are already present in this repo
+                # ToDo (pickles-bread-and-butter): remove all exceptions from the repo
```

Comment:
@pickles-bread-and-butter let me ask if you would remove this comment - mainly I think we don't really want to change the existing behavior of the repo. Jay (or perhaps it was me) wrote it the way they wanted to write it and it is what it is (for better or worse). Our job is just to make it work when FUSE-mounting the dataset.